docs/user/accumulo/install.rst

```diff
@@ -251,11 +251,8 @@ and installing `the WPS plugin`_.
         export CATALINA_OPTS="-Xmx8g -XX:MaxPermSize=512M -Duser.timezone=UTC \
         -server -Djava.awt.headless=true"
 
-    The value of ``-Xmx`` should be as large as your system will permit; this
-    is especially important for the Kafka plugin. You
-    should also consider passing ``-DGEOWEBCACHE_CACHE_DIR=/tmp/$USER-gwc``
-    and ``-DEPSG-HSQL.directory=/tmp/$USER-hsql``
-    as well. Be sure to restart Tomcat for changes to take place.
```

Comment:
Are we removing these suggestions for any particular reason?

Reply:
Nevermind me.

Reply:
moved them to https://github.com/locationtech/geomesa/pull/2324/files#diff-d697f43761355882cd7584f0891071dfR150 to make them more discoverable.